package-scripts.yml

```diff
@@ -0,0 +1,48 @@
+scripts:
```

Comment:
Haha, I didn't think we'd need to have this project use a yml file. The js version should be sufficient.